### PR TITLE
refactor: use wasabee portal

### PR DESCRIPTION
**Pull request by le-jeu**
don't merge yet
this is WIP to remove iitc portal uses as early as possible inside the source tree

 - [x] Find _almost_ all the occurrences 
 - [x] `Wasabee.get/fromIITC`
 - [x] `getAllPortalsOnScreen`
 - [x] blocker list
 - [x] ~Dlist~
 - [x] ~quickdraw~
 - [x] ~crosslink~

### CODE REVIEW

File: src/code/portal.js [hunk 1]
```diff
@@ -37,6 +37,33 @@ export default class WasabeePortal {
...
+          (data.lngE6 / 1e6).toFixed(6).toString()
+        );
+      }
+      // do we have enough to fake it?
+      if (data.latE6) {
+        return WasabeePortal.fake(
+          (data.latE6 / 1e6).toFixed(6).toString(),
+          (data.lngE6 / 1e6).toFixed(6).toString(),
+          id
+        );
```
Comment (cloudkucooland) [inline src/code/portal.js:60]: Need to set a [Loading:{id}] title here.

File: src/code/portal.js [hunk 2]
```diff
@@ -65,6 +92,12 @@ export default class WasabeePortal {
     return new L.LatLng(parseFloat(this.lat), parseFloat(this.lng));
   }
 
+  get team() {
```
Comment (cloudkucooland) [inline src/code/portal.js:95]: I like this

File: src/code/portal.js [hunk 3]
```diff
@@ -115,27 +144,7 @@ export default class WasabeePortal {
...
-    if (window.portals[id] && window.portals[id].options.data.latE6) {
-      const data = window.portals[id].options.data;
-      return WasabeePortal.fake(
-        (data.latE6 / 1e6).toFixed(6).toString(),
-        (data.lngE6 / 1e6).toFixed(6).toString(),
-        id
-      );
-    }
-    // nothing to get
-    return null;
```
Comment (cloudkucooland) [inline src/code/portal.js:138]: what does this do if window.portals[id] is undefined?

File: src/code/uiCommands.js [hunk 4]
```diff
@@ -175,7 +175,7 @@ export const getAllPortalsOnScreen = function(operation) {
         )
       )
         continue;
-      x.push(window.portals[portal]);
+      x.push(WasabeePortal.fromIITC(window.portals[portal]));
```
Comment (cloudkucooland) [inline src/code/uiCommands.js:178]: good

File: src/code/dialogs/blockersList.js [hunk 5]
```diff
@@ -199,12 +199,7 @@ const BlockerList = WDialog.extend({
 
     // add marker
     let type = window.plugin.wasabee.static.constants.MARKER_TYPE_DESTROY;
-    if (
-      window.portals[portalId] &&
-      window.portals[portalId].options &&
-      window.portals[portalId].options.data &&
-      window.portals[portalId].options.data.team == "E"
-    ) {
+    if (wportal.team == "E") {
```
Comment (cloudkucooland) [inline src/code/dialogs/blockersList.js:202]: I like the team static in there.

File: src/code/portal.js (hunk 1 above)
Comment (cloudkucooland) [reply src/code/portal.js:60]: (no square bracket, must match("^Loading: .*")

Comment (le-jeu) [reply src/code/portal.js:64]: `Loading` is locale dependent so I delegate this to a dynamic name that add this when we need the name to be display

File: src/code/portal.js (hunk 3 above)
Comment (le-jeu) [reply src/code/portal.js:138]: same as before, it returns  `null`

**Pull request merged**